### ISSUE TITLE
Progress Bar Does Not Appear in Jupyter Notebook

### ISSUE THREAD

**Issue by dalekube** [rstudio/keras#873]
The progress bar does not appear when fitting a model using Keras in an R kernel in Jupyter Notebook with default settings. The bar does not appear despite setting the 'verbose' parameter to 1 and 2.

R: 3.4.4
TensorFlow GPU-build: 1.14
Keras Package: 2.2.4.1
Jupyter Notebook: 5.7.8
IRkernel package for R kernel in Jupyter
Chrome (76.0.3809.132) and Internet Explorer (11.3085.14393.0)
fit.keras.engine.training.Model {keras}

Sample Chunk:
`# Fit the model
nn.model = model %>% fit(
  train_x,
  train_y,
  batch_size=250,
  shuffle=T,
  epochs=3,
  validation_data=list(test_x,test_y),
  verbose=1
  )`

**Comment by dalekube** [comment]
https://github.com/IRkernel/IRkernel/issues/416
